src/components/App/App.js

```diff
@@ -6,103 +6,93 @@ import {
   toggleActiveKeydown,
 } from '../../util/activateKeyUtil';
 import keyboardSwitch from '../../util/keyboardSwitch';
-import createSynth from '../SynthEngine/SynthEngine';
 
+import createSynth from '../SynthEngine/SynthEngine';
 import Scene from '../Scene/Scene';
 import Keyboard from '../Keyboard/Keyboard';
+import EffectKnob from '../EffectKnob/EffectKnob';
+import EffectToggle from '../EffectToggle/EffectToggle';
 
 const { oscillators, delay, reverb, filter } = createSynth();
 const engine = oscillators.chain(delay, reverb, filter, Tone.Destination);
 
 export default function App() {
-  const [synth, setSynth] = useState(engine);
+  const [synth] = useState(engine);
   const [detune, setDetune] = useState(0);
-  const [osc, setOsc] = useState(synth.get().oscillator.type);
-
-  const triggerKeyDownPlay = async (e) => {
-    if (e.type === 'keydown' && keyboardSwitch(e)) {
-      await Tone.start();
-      const note = keyboardSwitch(e);
-      synth.triggerAttackRelease(note, '8n');
-      toggleActiveKeydown(note);
-      console.log(note);
-      return;
-    }
-  };
-
-  const triggerOnClickPlay = async (e) => {
-    if (e.target.attributes.note) {
-      await Tone.start();
-      const note = e.target.attributes.note.value;
-      synth.triggerAttackRelease(note, '8n');
-      toggleActiveClick(e);
-      console.log(note);
-      return;
-    }
-  };
+  const [cutoff, setCutoff] = useState(filter.get().frequency);
+  const [oscType, setOscType] = useState(synth.get().oscillator.type);
 
   useEffect(() => {
+    const triggerKeyDownPlay = async (e) => {
+      if (e.type === 'keydown' && keyboardSwitch(e)) {
+        await Tone.start();
+        const note = keyboardSwitch(e);
+        synth.triggerAttackRelease(note, '8n');
```

Comment:
This 8n is referencing the duration of the note. This could potentially be varied by an encoder as a decay knob.